projects/exui-common-lib/src/lib/directives/manage-link-toggle/manage-link-toggle.directive.ts

```diff
@@ -0,0 +1,18 @@
+import {Directive, ElementRef, Input, OnInit} from '@angular/core';
+
+@Directive({
+  selector: '[xuilibManageLinkToggle]'
+})
+export class ManageLinkToggleDirective implements OnInit {
+  @Input() public hidden: boolean = true;
+
+  constructor(private readonly element: ElementRef) {
+  }
+
+  public ngOnInit(): void {
+    const host = this.element.nativeElement as HTMLElement;
+    if (this.hidden) {
+      host.style.display = 'none';
+    }
```

Comment:
Is this all the business logic for just hide an element?

Reply:
apparently yes